applications/vanilla/views/post/discussion.php

```diff
@@ -44,7 +44,7 @@
     $this->fireEvent('BeforeBodyInput');
 
     echo '<div class="P">';
-    echo $this->Form->bodyBox('Body', ['Table' => 'Discussion', 'FileUpload' => true]);
+    echo $this->Form->bodyBox('Body', ['Table' => 'Discussion', 'FileUpload' => true, 'placeholder' => t('Type your message'), 'title' => t('Type your message')]);
```

Comment:
"message" is a term normally associated with conversations.

Reply:
I had checked with @tburry. We have multiple types of posts that all use this same string, so message seemed like a good neutral word for all of these post types.

Reply:
Message is fine in this context.